### DISCUSSION TITLE
AlwaysDisplayed

### DISCUSSION THREAD

**Discussion by i-make-robots** [andrewauclair/ModernDocking#206] [Q&A]
I'm looking for a panel that sits behind all the others and doesn't have a tab.  Is this the "alwaysDisplayed" example?
It would be very handy for Wiki to show a screenshot of each demo, much like https://docs.oracle.com/javase/tutorial/uiswing/layout/visual.html

**Comment by andrewauclair** [comment]
I'm not sure what you mean by behind. Do you mean other panels around the root?

You can add the root panel to any other panel and layout manager. I do this at work where I have a status panel below the root panel.

**Reply by i-make-robots** [reply]
I mean like in IntelliJ there's text visible when there are no docked panels: 
![image](https://github.com/user-attachments/assets/d59c101e-f0ae-4512-b614-1e34923702ce)
